ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -31,6 +31,7 @@ operator:
   - as: wmco-bundle
     context_dir: .
     dockerfile_path: bundle.Dockerfile
+    optional: true
```

Comment:
how is this deprecated?

Reply:
This isn't building an index, but a bundle https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#building-operator-bundles

Reply:
@sebsoto If the bundle is not getting built now how are the tests that depend on it running and passing?

Reply:
To be honest I'm a bit confused here. I'm not following where/what caused this to not be built.